docs/src/components/propsTable/PropsTableBody.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react';
+import { remark } from 'remark';
+import html from 'remark-html';
+import { Badge, TableRow, View } from '@aws-amplify/ui-react';
+import { ResponsiveTableCell } from '@/components/ResponsiveTable';
+import { CodeHighlight } from '@/components/CodeHighlight';
+import { Property } from '../../../scripts/types/catalog';
+
+export const PropsTableBody = ([
+  propName,
+  { name, type, description, isOptional },
+]: [string, Property]): JSX.Element => {
+  const [htmlDescription, setHtmlDescription] = useState('-');
+
+  useEffect(() => {
+    const getDesc = async () => {
+      const descriptionHTML = await remark().use(html).process(description);
```

Comment:
`description` is a string of markdown right? Could we process the markdown at build time rather than on the client here? Is there a benefit to processing the markdown in the client?

Reply:
Good point. I didn't think about it. Let me try that.

Reply:
I changed to [`react-markdown`](https://www.npmjs.com/package/react-markdown) instead. It's still in client, but looks a lot better. Not sure how to process it at build time?

Reply:
This is good enough. If we wanted to compile at build time you could put the `remark().use(html).process(description)` in your script that generates the JSON so the `description` attribute in the JSON is raw HTML. But I think this is fine